packages/esm-patient-chart-app/src/hooks/useContextWindowSize.tsx

```diff
@@ -6,7 +6,7 @@ import { useWorkspace } from '../hooks/useWorkspace';
 interface WindowSize {
   size: string;
 }
-type ActionTypes = 'minimize' | 'maximize' | 'hide' | 'reopen';
+type ActionTypes = 'minimize' | 'maximize' | 'hide' | 'reopen' | string;
```

Comment:
These types are a collection of fixed strings. If you make it `| string` it will be effectively the same as `type ActionTypes = string`, which defeats its purpose.